lib/Bio/EnsEMBL/DataCheck/Checks/CheckSyntenySanity.pm

```diff
@@ -0,0 +1,106 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckSyntenySanity;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckSyntenySanity',
+  DESCRIPTION    => 'Check for missing syntenies in the compara database',
+  GROUPS         => ['compara', 'compara_syntenies'],
+  DATACHECK_TYPE => 'advisory',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['dnafrag', 'dnafrag_region', 'genome_db', 'genomic_align', 'method_link', 'method_link_species_set', 'synteny_region']
+};
+
+sub tests {
+  my ($self) = @_;
+  # Make sure the registry is loaded
+  $self->registry;
```

Comment:
I wouldn't have thought that this would be necessary - I realise it was a while ago now, but if you could remember why you needed it, that would be helpful, as that might be something I need to take a look at.

Reply:
It is not necessary, I think that is a line I forgot to remove when I was running some early tests, I will take it away!